README.md

```diff
@@ -48,9 +48,9 @@ Installation and documentation
 ------------------------------
 
 - Full documentation is available on [Docker's website](https://docs.docker.com/compose/).
-- If you have any questions, you can talk in real-time with other developers in the #docker-compose IRC channel on Freenode. [Click here to join using IRCCloud.](https://www.irccloud.com/invite?hostname=irc.freenode.net&channel=%23docker-compose)
-- Code repository for Compose is on [GitHub](https://github.com/docker/compose)
-- If you find any problems please fill out an [issue](https://github.com/docker/compose/issues/new)
+- If you have any questions, you can talk in real-time with other developers in the `#docker-compose` IRC channel on Freenode. [Click here to join using IRCCloud](https://www.irccloud.com/invite?hostname=irc.freenode.net&channel=%23docker-compose).
```

Comment:
I don't know how active this is. It might make sense to remove this entirely.

Reply:
Right now it's 40 nicks, 0 ops. Are you undecided or decided for removal? Please let me know whether you want me to drop IRC or not.

Reply:
Let's remove it as I don't think we have any official presence there anymore. Thanks 👍

Reply:
Done